crates/driver/src/infra/solver/dto/solution.rs

```diff
@@ -223,14 +223,14 @@ impl Solutions {
 
 #[serde_as]
 #[derive(Debug, Deserialize)]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
Should we keep it from the solver's side to avoid redundant data from the third-party solvers' side?